main/boards/waveshare-s3-epaper-1.54/README.md

```diff
@@ -31,18 +37,18 @@ idf.py menuconfig
 **选择板子**
 
 ```bash
-Xiaozhi Assistant -> Board Type -> Waveshare ESP32-S3-ePaper-1.54
+Xiaozhi Assistant -> Board Type -> Waveshare ESP32-S3-ePaper-1.54_v2
 ```
 
 **编译**
 
-```ba
-idf.py build
+```bash
+python ./scripts/release.py --name waveshare-s3-epaper-1.54-v2 waveshare-s3-epaper-1.54
```

Comment:
idf.py menuconfig does not write the additional sdk options from config.json to sdkconfig, the release.py script does and calls idf.py build, so use that script